### PR TITLE
Feat add introduction page

### PR DESCRIPTION
**Pull request by nerdyspook**
**Added pages:**
-  Introduction
- Colors
- Typography
- Conclusion

intro

### CODE REVIEW

File: styles/documentation.css [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+.content-details {
+    color: var(--text-color-medium);
+}
+
+@media screen and (min-width: 768px) {
+    .content {
+        width: 73%;
+        margin: var(--mb-2-5);
+        margin-left: calc(20% + 3rem);
+        padding: 0.25rem;
```
Comment (Prabhav-Sharma) [inline styles/documentation.css]: You can also use global variables for padding

**Review by Prabhav-Sharma — COMMENTED** [review]
Try to only change files pertaining to the component. Create separate PRs for other files

File: styles/documentation.css (hunk 1 above)
Comment (nerdyspook) [reply styles/documentation.css]: Thanks, I have made the changes you suggested.

**Review by SukanyaSen530 — APPROVED** [review]
Looks good :)

File: styles/documentation.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+}
+.dark-theme .example-container,
+.dark-theme .examples-container {
+    color: var(--text-color);
+    background-color: var(--secondary-color-alt);
+}
+
+.hidden-code {
+    color: var(--default-color);
+    background-color: #feebc8;
```
Comment (KanekarSnehal) [inline styles/documentation.css:70]: you can use css variable here also

**Review by KanekarSnehal — APPROVED** [review]
Rest all looks good you can merge it.👍

File: styles/documentation.css (hunk 2 above)
Comment (nerdyspook) [reply styles/documentation.css:70]: I haven't used this color anywhere else, so I thought I can just declare it here.

**Pull request merged**